ui/ozone/platform/starboard/surface_factory_starboard.cc

```diff
@@ -16,8 +16,27 @@
 
 namespace ui {
 
-SurfaceFactoryStarboard::SurfaceFactoryStarboard() {}
+SurfaceFactoryStarboard::SurfaceFactoryStarboard() {
+  egl_implementation_ = std::make_unique<GLOzoneEGLStarboard>();
+}
 
 SurfaceFactoryStarboard::~SurfaceFactoryStarboard() {}
 
+std::vector<gl::GLImplementationParts>
+SurfaceFactoryStarboard::GetAllowedGLImplementations() {
+  return std::vector<gl::GLImplementationParts>{
+      gl::GLImplementationParts(gl::kGLImplementationEGLGLES2),
+  };
+}
+
+GLOzone* SurfaceFactoryStarboard::GetGLOzone(
+    const gl::GLImplementationParts& implementation) {
+  switch (implementation.gl) {
+    case gl::kGLImplementationEGLGLES2:
+    case gl::kGLImplementationEGLANGLE:
```

Comment:
l.28 doesn't list kGLImplementationEGLANGLE as an allowed implementation, so we should not handle it here IIUC.

Reply:
Done, but we may need to look at supporting that for Linux development. IIRC I needed that line for the prototype to work on my machine.

Reply:
Looping back around to this, I applied this to the prototype and it required another hacky workaround (for now, looking into getting actual solutions to the 2 existing ones and this new one) to avoid it setting the the gl implementation to ANGLE in gl_display.cc, regardless of the flags I tried. It'll probably be a few cherry picks once I can find where that's fixed.